lib/set.js

```diff
@@ -1,5 +1,15 @@
-var set = function(data) {
-  this.data = data;
+var _ = require('lodash');
+
+var set = function(path, data) {
+  if (!data){
```

Comment:
I think there is a space missing before the starting of braces
`if (!data) {`
Linter should have detected this!!

Reply:
i have to change in eslint configuration.

Reply:
I will fix this in next PR